lib/github-checks.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const chalk = require('chalk');
+const execa = require('execa');
+const ghGot = require('gh-got');
+const hostedGitInfo = require('hosted-git-info');
+const pTimeout = require('p-timeout');
+const pWaitFor = require('p-wait-for');
+const terminalLink = require('terminal-link');
+const util = require('./util');
+
+const THIRTY_SECONDS = 1000 * 30;
+const THIRTY_MINUTES = 1000 * 60 * 30;
+
+module.exports = async task => {
+	const hasUnpushedCommits = await execa.stdout('git', ['cherry']) !== '';
```

Comment:
We're starting to get a *lot* of Git related command now. Would be nice to move these to a `git` util kinda file with various utility methods.

```js
if (await git.hasUnpushedCommits) {

}
```

Reply:
https://github.com/sindresorhus/np/blob/master/lib/git-util.js

Reply:
@sindresorhus Since np uses quite a bit of npm-related (and yarn-related) commands, do you think we should move those to a separate `npm-util.js` file as well?

Reply:
Yes. PR welcome for that :)

Reply:
Note that this already exists (see `source/git/util.js`). You can move this to there.

Reply:
I’ve moved `hasUnpushedCommits` to `git-util.js` in a39de24bd3bcb30563fd789e124dafa307b78949.